go.mod

```diff
@@ -88,8 +88,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.11.2
-	istio.io/api v0.0.0-20230411172800-cb047abbdf91
-	istio.io/client-go v1.18.0-alpha.0.0.20230411173201-f27e09ad6e63
+	istio.io/api v0.0.0-20230406193325-c74882c8130f
```

Comment:
revert?

Reply:
As noted in the channel, one needs to be extra vigilant with update_deps or any job which is simply doing a `go get`. As noted here, you can see the data is moving back which is indicative of an issue.